xml/tuning_cgroups.xml

```diff
@@ -121,13 +121,24 @@
      <para>
       Unused memory from processes below this limit will not be
       reclaimed for other use. Use suffixes K, M, G or T for
-      <replaceable>BYTES</replaceable>. This requires
+      <replaceable>BYTES</replaceable>. This implies
       <literal>MemoryAccounting=yes</literal>.
      </para>
      <para>
       Example:
      </para>
      <screen>&prompt.root;<command>systemctl set-property nginx.service MemoryLow=512M</command></screen>
+     <note>
+      <title>Unified Control Group Hierarchy</title>
+      <para>
+       This setting is supported only if the unified control group hierarchy is
```

Comment:
I'd suggest s/supported/available/ or s/supported/possible/, i.e. make the documentation independent of support status.